frontend/src/individual_edit.tsx

```diff
@@ -229,208 +232,223 @@ export function IndividualEdit({id}: {id: string | undefined}) {
+                      </a>]
+                    </span>
+                  </li>)}
+              </ul>
+              <div className={style.flexRow}>
+                <KeyboardDatePicker
+                    autoOk
+                    variant="inline"
+                    className={style.control}
+                    inputVariant={inputVariant}
+                    label="Mätningsdatum"
+                    format={dateFormat}
+                    value={weightDate}
+                    InputLabelProps={{
+                      shrink: true,
+                    }}
+                    onChange={(date, value) => {value && setWeightDate(value)}}
+                  />
+                <TextField label='Vikt' className={style.control}
+                  value={weight}
+                  type='number'
+                  variant={inputVariant}
+                  InputProps={{
+                    endAdornment: <InputAdornment position='end'>Kg</InputAdornment>
+                  }}
+                  InputLabelProps={{
+                    shrink: true,
+                  }}
+                  onChange={(e: any) => {setWeight(e.target.value)}}
+                  />
+              </div>
 
-            <Button variant="contained"
-                    color="primary"
-                    onClick={() => {updateField('weights', [...individual.weights, {date: weightDate, weight: weight}])}}>
-              {'Lägg till viktmätning'}
-            </Button>
-            <h3>Hull</h3>
-            <ul>
-              {individual.bodyfat && individual.bodyfat.map((b: DateBodyfat, i: number) =>
-                <li key={i} className={style.measureList}>
-                  {`${new Date(b.date).toLocaleDateString('sv')} - ${b.bodyfat}`}
-                  <span className={style.listButton}>
-                    [<a className={style.scriptLink} onClick={() => removeMeasure('bodyfat', i)}>
-                      Delete
-                    </a>]
-                  </span>
-                </li>)}
-            </ul>
-            <div className={style.flexRow}>
-              <TextField label='Mätningsdatum' className={style.control}
-                value={hullDate}
-                type='date'
-                variant={inputVariant}
-                InputLabelProps={{
-                  shrink: true,
-                }}
-                onChange={(e: any) => {setHullDate(e.target.value)}}
+              <Button variant="contained"
+                      color="primary"
+                      onClick={() => {updateField('weights', [...individual.weights, {date: weightDate, weight: weight}])}}>
+                {'Lägg till viktmätning'}
+              </Button>
+              <h3>Hull</h3>
+              <ul>
+                {individual.bodyfat && individual.bodyfat.map((b: DateBodyfat, i: number) =>
+                  <li key={i} className={style.measureList}>
+                    {`${new Date(b.date).toLocaleDateString(locale)} - ${b.bodyfat}`}
```

Comment:
Maybe just extract `d => new Date(d).toLocaleDateString(locale)` to a separate function

Reply:
seems fair! :)

Reply:
Yes this seems to work well when I test so perhaps address Dan's comments and its good to go